### PR TITLE
function schedules should not use `oneshotTokenExchange`

### PR DESCRIPTION
**Pull request by haakonvt**
# Description

Please describe the change you have made.

## Checklist

- [ ] Tests added/updated.
- [ ] Run Demo Job Locally.
- [ ] Documentation updated.
- [x] Changelogs updated in [CHANGELOG.cdf-tk.md](https://github.com/cognitedata/cdf-project-templates/blob/main/CHANGELOG.cdf-tk.md).
- [x] Template changelogs updated in [CHANGELOG.templates.md](https://github.com/cognitedata/cdf-project-templates/blob/main/CHANGELOG.templates.md).
- [ ] Version bumped.
  [_version.py](https://github.com/cognitedata/cdf-project-templates/blob/main/cognite/cognite_toolkit/_version.py) and
  [pyproject.toml](https://github.com/cognitedata/cdf-project-templates/blob/main/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

File: cognite_toolkit/_cdf_tk/load/_resource_loaders.py [hunk 1]
```diff
@@ -638,50 +637,26 @@ def retrieve(self, ids: SequenceNotStr[str]) -> FunctionSchedulesList:
...
+                    description=item.description or "",
+                    cron_expression=cast(str, item.cron_expression),
+                    function_id=cast(int, item.function_id),
+                    data=item.data,
+                    client_credentials=client_credentials,
                 )
-            except CogniteAPIError as e:
-                if e.code == 400 and "Failed to bind session" in e.message:
-                    print("  [bold yellow]WARNING:[/] Failed to bind session because function is not ready.")
-                continue
```
Comment (haakonvt) [inline cognite_toolkit/_cdf_tk/load/_resource_loaders.py:681]: Reason for removing: The session binds just fine for a function that is still deploying. I guess the bind fails if an invalid nonce is passed which was possible before, see line (also removed):
`nonce = session.nonce if session is not None else ""`

**Review by doctrino — COMMENTED** [review]
Great if you can update the changelog as well

```markdown
## TBD
### Fixed
* My fantastic fix of session revoke issue for client schedule.
```

Will likely have to do a sherry picked release for this one as well.

File: tests/tests_unit/approval_client/client.py [hunk 2]
```diff
@@ -428,6 +430,13 @@ def create_function_api(**kwargs) -> Function:
             created_resources[resource_cls.__name__].append(created)
             return Function.load(created.dump(camel_case=True))
 
+        def create_function_schedule_api(**kwargs) -> FunctionSchedule:
+            # Function API does not follow the same pattern as the other APIs
+            # So needs special handling
```
Comment (haakonvt) [inline tests/tests_unit/approval_client/client.py:435]:
```suggestion
```

**Review by doctrino — APPROVED** [review]
Great job.

**Pull request merged**